### PR TITLE
assignment2

### PR DESCRIPTION
**Pull request by hunter097**

### CODE REVIEW

File: Assignment2/hunter097-1.b.ttl [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+	
+:Sensor029 a rdfs:Class;
+	rdfs:subClassOf:Class.
+
+:Computer101 a rdfs:Class;
+	rdfs:subClassOf:Class.
+
+Measurement8401:hasMeasurement a rdf:Sensor029;
+	rdfs:hasTemperature:29;
+	rdfs:atTime:2010-06-12T 12:00:12.
```
Comment (ocorcho) [inline Assignment2/hunter097-1.b.ttl:15]: Problem with syntax. This will not parse

File: Assignment2/hunter097-1.b.ttl [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+User10A:hasOwner a rdf:Computer101;
+	rdfs:hasName:Pedro.
```
Comment (ocorcho) [inline Assignment2/hunter097-1.b.ttl:18]: Same here

**Review by ocorcho — CHANGES_REQUESTED** [review]

File: Assignment2/hunter097-1.b.ttl (hunk 1 above)
Comment (hunter097) [reply Assignment2/hunter097-1.b.ttl]: I think I fixed the error

File: Assignment2/hunter097-1.b.ttl [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+:Class a rdfs:Class.
+	
+:Sensor029 a rdfs:Class;
+	rdfs:subClassOf:Class.
+
+:Computer101 a rdfs:Class;
+	rdfs:subClassOf:Class.
+
+:Measurement8401:hasMeasurement a rdf:Sensor029;
+	rdfs:hasTemperature:"29"^^xsd:integer;
```
Comment (ocorcho) [inline Assignment2/hunter097-1.b.ttl:14]: Wrong syntax still. Check other examples from other students, already accepted, and you will realise about them.

**Review by ocorcho — CHANGES_REQUESTED** [review]

File: Assignment2/hunter097-1.b.ttl (hunk 3 above)
Comment (hunter097) [reply Assignment2/hunter097-1.b.ttl]: i was checking other examples, i think the error was in the ":" of line 14

File: Assignment2/hunter097-1.b.ttl [hunk 4]
```diff
@@ -0,0 +1,18 @@
+@base < www.oeg upm.net / person >
```
Comment (ocorcho) [inline Assignment2/hunter097-1.b.ttl:1]: Prefixes should point to http:// URI paths.

File: Assignment2/hunter097-1.b.ttl [hunk 5]
```diff
@@ -0,0 +1,18 @@
+@base < www.oeg upm.net / person >
+@prefix : < www.oeg upm.net / person #>
+@prefix person : < www.oeg upm.net / person #>
+
+:Class a rdfs:Class.
+	
+:Sensor029 a rdfs:Class;
+	rdfs:subClassOf:Class.
```
Comment (ocorcho) [inline Assignment2/hunter097-1.b.ttl:8]: Wrong syntax.

File: Assignment2/hunter097-1.b.ttl [hunk 6]
```diff
@@ -0,0 +1,18 @@
...
+@prefix : < www.oeg upm.net / person #>
+@prefix person : < www.oeg upm.net / person #>
+
+:Class a rdfs:Class.
+	
+:Sensor029 a rdfs:Class;
+	rdfs:subClassOf:Class.
+
+:Computer101 a rdfs:Class;
+	rdfs:subClassOf:Class.
```
Comment (ocorcho) [inline Assignment2/hunter097-1.b.ttl:11]: Wrong syntax and bad use of prefix rdfs. Same below.

**Review by ocorcho — CHANGES_REQUESTED** [review]

File: Assignment2/hunter097-1.b.ttl (hunk 6 above)
Comment (hunter097) [reply Assignment2/hunter097-1.b.ttl]: I think undertood

File: Assignment2/hunter097-1.b.ttl [hunk 7]
```diff
@@ -0,0 +1,13 @@
...
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#>
+
+:Class01:includes:Sensor029.
+:Class01:includes:Computer101.
+
+:Measurement8401:hasMeasurement:Sensor029;
+	Sensor029:hasTemperature "29"^^xsd:integer;
+	Sensor029:atTime "2010-06-12T12:00:12"^^xsd:dateTime.
+
+:Computer101:hasOwner:User10A;
```
Comment (ocorcho) [inline Assignment2/hunter097-1.b.ttl:12]: Wrong syntax. Does not parse

**Review by ocorcho — CHANGES_REQUESTED** [review]

File: Assignment2/hunter097-1.b.ttl (hunk 7 above)
Comment (ocorcho) [reply Assignment2/hunter097-1.b.ttl:12]: Still wrong syntax. Can you please validate with an RDF parser before?

**Review by ocorcho — CHANGES_REQUESTED** [review]

**Comment by ocorcho** [comment]
Exercise 2a is correct. Now @paoespinozarias will check assignment 2b

File: Assignment2/hunter097-fromJSONtoJSON-LD.json [hunk 8]
```diff
@@ -0,0 +1,60 @@
...
+    "properties:subjects":[
+        {
+            "properties:subjectName":"Redes de Computadores",
+            "properties:teachers":[
+                "Francisco Javier Soriano",
+                "Genoveva Lopez Gomez",
+                "Victor Ramperez Martin"
+            ],
+            "properties:description":"El objetivo de esta asignatura es comprender el funcionamiento de las tecnologías básicas de comunicaciones que constituyen el núcleo de Internet, y de las redes de empresas y organizaciones."
+        }
```
Comment (paoespinozarias) [inline Assignment2/hunter097-fromJSONtoJSON-LD.json:23]: Add a comma at the end.

File: Assignment2/hunter097-fromJSONtoJSON-LD.json [hunk 9]
```diff
@@ -0,0 +1,60 @@
+{
+    "@context":{
+        "xsd":"http://www.w3.org/2001/XMLSchema#",
+        "propieties": "http://www.examples.org/properties/"
+    },
+    "properties:name":"Carlos Martin Sanchez",
```
Comment (paoespinozarias) [inline Assignment2/hunter097-fromJSONtoJSON-LD.json:6]: Please, define a type and id.

**Review by paoespinozarias — CHANGES_REQUESTED** [review]

File: Assignment2/hunter097-fromRDFtoJSON-LD.json [hunk 10]
```diff
@@ -0,0 +1,49 @@
...
+        },
+        {
+            "@id":"propieties:User10A",
+            "propieties:hasName": {
+                "@type":"xsd:normalizedstring",
+                "@value":"Pedro"
+            }
+        }
+    ]
+}
```
Comment (paoespinozarias) [inline Assignment2/hunter097-fromRDFtoJSON-LD.json:49]: This file is ok, well done!

**Review by paoespinozarias — APPROVED** [review]

**Review by paoespinozarias — APPROVED** [review]

**Pull request merged**